src/geo/transform.js

```diff
@@ -20,6 +20,14 @@ import type {PaddingOptions} from './edge_insets.js';
 const NUM_WORLD_COPIES = 3;
 const DEFAULT_MIN_ZOOM = 0;
 
+const elevationSamplePoints = [
```

Comment:
Yeah, it was entirely to avoid reinstantiating the array, but maybe relative to the number of temporary small-vector/matrix allocations per frame, this sort of concern is unreasonable.

Reply:
I've move it inside the function 🤷